src/components/Navbar.tsx

```diff
@@ -0,0 +1,90 @@
+import LoginBtn from './LoginBtn';
+import { useTranslation } from 'next-i18next';
+import type { GetStaticProps, InferGetStaticPropsType } from 'next';
+import { getStaticProps } from '@/pages';
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
+
+ //lines below will be needed fo authentication
+// interface _Props {
+//   authData: {
+//     user?: {
+//       name?: string | null | undefined;
+//       email?: string | null | undefined;
+//       image?: string | null | undefined;
+//     };
+//   } | null;
+//   authStatus: 'authenticated' | 'loading' | 'unauthenticated';
+// }
+
+const Navbar = function (
+  _Props: InferGetStaticPropsType<typeof getStaticProps>,
+  //lines below will be needed fo authentication
+  // authData: {
+  //   user?: {
+  //     name?: string | null | undefined;
+  //     email?: string | null | undefined;
+  //     image?: string | null | undefined;
+  //   };
+  // } | null,
+  // authStatus?: 'authenticated' | 'loading' | 'unauthenticated' | 'undefined'
+) {
+  const { t } = useTranslation('navbar');
+
+  return (
+    <div className="navbar bg-[#abd9e1] px-2 shadow-xl">
+      <div className="logo flex-1">
+        <a className="btn-ghost btn text-xl normal-case">
+          <img
+            alt="Journey الرحلة logo"
+            src="https://media1-production-mightynetworks.imgix.net/asset/56053552/JOURNEYUNCROPPED2-28.png?ixlib=rails-4.2.0&amp;auto=format&amp;w=150&amp;h=100&amp;fit=crop&amp;impolicy=Avatar&amp;crop=faces"
+            width="60"
+          />
```

Comment:
❌ 
use [nextjs image component](https://nextjs.org/docs/app/building-your-application/optimizing/images) instead of plain html <img /> tag

Reply:
Done